src/App.tsx

```diff
@@ -1,24 +1,146 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Eror } from './components/ERROR';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/FilterEnum';
 
-const USER_ID = 0;
+const USER_ID = 10387;
+
+export enum Error {
+  LOAD = 'load',
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [title, setTitle] = useState<string>('');
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const visibleTodos = todos.filter((todo) => {
+    switch (filterBy) {
+      case FilterBy.Active:
+        return !todo.completed;
+
+      case FilterBy.Completed:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  const addError = (error: Error) => {
```

Comment:
Consider renaming the function 'addError' to 'displayError' for better clarity of its purpose.